### PR TITLE
fix: more informative logs

### LINKED ISSUE #68: Better logging and grouping logs
Currently the logs are not providing all neccessary debug information. Also it's hard to track which information came from which request. 

Logs require cleanup, more information and better grouping.

### PR DESCRIPTION
**Pull request by Rakowskiii**
# Description

Improve logs for better visibility of data and easier debugging.

Resolves #68 

## How Has This Been Tested?

Ran locally.

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/handlers/notify.rs [hunk 1]
```diff
@@ -110,7 +107,7 @@ async fn process_publish_jobs(
     jobs: Vec<PublishJob>,
     client: Arc<relay_client::websocket::Client>,
     response: &mut Response,
-    request_id: uuid::Uuid,
+    uuid: uuid::Uuid,
```
Comment (arein) [inline src/handlers/notify.rs:110]: Up to you - personally find `request_id` more mnemonic

File: src/handlers/notify.rs [hunk 2]
```diff
@@ -134,13 +131,10 @@ async fn process_publish_jobs(
                 response.sent.insert(account.to_string());
```
Comment (arein) [inline src/handlers/notify.rs:131]: Need the topic and ideally `rpc_id` of the relay message here

**Review by arein — CHANGES_REQUESTED** [review]

File: src/handlers/notify.rs (hunk 1 above)
Comment (HarryET) [reply src/handlers/notify.rs:110]: `uuid` also has collisions with the package name

File: src/handlers/webhooks/register_webhook.rs [hunk 3]
```diff
@@ -23,7 +23,8 @@ pub async fn handler(
     State(state): State<Arc<AppState>>,
     Json(webhook_info): Json<WebhookConfig>,
 ) -> Result<impl IntoResponse> {
-    info!("Registering webhook for project: {}", project_id);
+    let uuid = uuid::Uuid::new_v4();
```
Comment (HarryET) [inline src/handlers/webhooks/register_webhook.rs:26]: could this be named better? same for a few more

**Review by HarryET — CHANGES_REQUESTED** [review]

File: src/handlers/notify.rs (hunk 2 above)
Comment (Rakowskiii) [reply src/handlers/notify.rs:136]: Added topic. `rpc_id` may be problematic as `relay client` is not sharing this data with consumer.
I'm currently working on moving from this to HTTP client. Will try to fix it with it if that's ok

Comment (arein) [reply src/handlers/notify.rs:136]: Yup cool to add later.

**Review by arein — APPROVED** [review]

**Review by HarryET — APPROVED** [review]

**Pull request merged**